### DISCUSSION TITLE
Documentation for @xstate/viz

### DISCUSSION THREAD

**Discussion by sawyerh** [statelyai/xstate#1279] [General] [answered]
An [NPM package exists for creating visualizations](https://github.com/davidkpiano/xstate/tree/master/packages/xstate-viz) similar to the [XState Visualizer tool](https://xstate.js.org/viz/), however currently lacks documentation. I managed to get this to work in a [prototype](https://github.com/sawyerh/xstate-flow-demo/blob/429eb104e5918146b2e7c8e255d0908a09ebdce4/src/components/FormStateEditor.js#L49), however ran into troubles loading the default styles, which don't seem like they're included in the version I installed. Having clearer documentation for the `@xstate/viz` package would be really helpful 🙏

**Comment by davidkpiano** [comment, marked as answer]
It's not meant to be publicly consumed yet (I'm developing it in the open) so I'm not going to put effort into documentation until it's ready to be released.

**Reply by sawyerh** [reply]
Got it, thanks @davidkpiano
